cmd/sourced/cmd/root.go

```diff
@@ -65,6 +67,12 @@ func log(err error) {
 		printRed("Cannot perform this action, full re-initialization is needed, run 'prune' command first")
 	case dir.ErrNotValid.Is(err):
 		printRed("Cannot perform this action, config directory is not valid")
+	case compose.ErrCouldNotGetComposer.Is(err):
+		printRed("docker-compose is not installed, and there was an error while trying to use the container alternative" +
+			"\nsee: https://docs.sourced.tech/community-edition/quickstart/1-install-requirements#docker-compose")
```

Comment:
Similar to the other comment, I think for most people it can be confusing to explain how to run things offline, when that is not the intended use case.
I'd prefer to remove the link to the docs, the error message should be enough to let the user understand that they need to connect to the internet, open their firewall, or provide the file manually.

Reply:
The linked docs do not explain how to run source{d} offline. They only explain that docker-compose is a requirement, and how can it be satisfied.
I think that pointing to docs from errors can be a good idea.